File: PantryMateAPI/PantryMate.API/Entities/Item.cs
using System.Collections.Generic;

namespace PantryMate.API.Entities
{
    public class Item : BaseEntity
    {
        public Item()
        {
            PantryItems = new List<PantryItem>();
        }

        public int ItemId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int AccountId { get; set; }
        public int? BrandId { get; set; }
        public decimal? Price { get; set; }
        public int? UnitOfMeasureId { get; set; }

        public virtual Brand Brand { get; set; }
        public virtual UnitOfMeasure UnitOfMeasure { get; set; }
        public virtual ICollection<PantryItem> PantryItems { get; set; }
    }
}

File: pantry-mate-web-app/src/app/models/item.ts
export class ItemFormModel {
    constructor ( public name = '', public price = '' ) {

    }
}

export interface Item {
    abbreviatedUnitOfMeasure: string;
    brand: string;
    description: string;
    itemId: number;
    name: string;
    price: number;
    unitOfMeasure: string;
}

export interface NewItemRequest {
    name: string;
    price: number;
    description: string;
    unitOfMeasureId?: number;
    brandId?: number;
}